posthog/temporal/batch_exports/http_batch_export.py

```diff
@@ -0,0 +1,251 @@
+import aiohttp
+import datetime as dt
+import json
+from dataclasses import dataclass
+
+from django.conf import settings
+from temporalio import activity, workflow
+from temporalio.common import RetryPolicy
+
+from posthog.batch_exports.service import BatchExportField, BatchExportSchema, HttpBatchExportInputs
+from posthog.temporal.batch_exports.base import PostHogWorkflow
+from posthog.temporal.batch_exports.batch_exports import (
+    BatchExportTemporaryFile,
+    CreateBatchExportRunInputs,
+    UpdateBatchExportRunStatusInputs,
+    create_export_run,
+    execute_batch_export_insert_activity,
+    get_data_interval,
+    get_rows_count,
+    iter_records,
+    json_dumps_bytes,
+)
+from posthog.temporal.batch_exports.clickhouse import get_client
+from posthog.temporal.batch_exports.metrics import (
+    get_bytes_exported_metric,
+    get_rows_exported_metric,
+)
+from posthog.temporal.common.logger import bind_temporal_worker_logger
+
+
+def http_default_fields() -> list[BatchExportField]:
+    return [
+        BatchExportField(expression="toString(uuid)", alias="uuid"),
+        BatchExportField(expression="timestamp", alias="timestamp"),
+        BatchExportField(expression="event", alias="event"),
+        BatchExportField(expression="nullIf(properties, '')", alias="properties"),
+        BatchExportField(expression="toString(distinct_id)", alias="distinct_id"),
+        BatchExportField(expression="toJSONString(elements_chain)", alias="elements_chain"),
```

Comment:
Do we need `toJSONString` for `elements_chain`?

This was used in other destinations as a bit of a backwards compatibility hack: Before batch exports, `elements_chain` was parsed into an object, and exported to a JSON field. With batch exports, doing that parsing was prohibitively expensive and we assumed most folks weren't using `elements_chain`. However, in order to ensure compatibility with existing JSON fields, we dumped it with `toJSONString`.

Reply:
Good point, and I guess the giveaway is that the migration script is just passing the string along: https://github.com/PostHog/posthog-migration-tools/blob/main/migrate.py#L360